### PR TITLE
fix about, contact, career

### PR DESCRIPTION
**Pull request by ebuka-nwabueze**

### CODE REVIEW

File: src/components/FeaturedEventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+const FeaturedEventCard: React.FC<FeaturedEventProp> = ({ event }) => {
+	const [panelOpen, setpanelOpen] = useState(false);
+
+	return (
+		<StyledWrapperDiv>
+			<StyledDetailsSection>
+				{/* left content */}
+				<StyledLeftContent>
+					<StyledCourseFormat type={event.event_format}>
+						{event.event_format}
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: use camelCase for your attributes

File: src/components/FeaturedEventCard.tsx [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+					</StyledCourseFormat>
+					<h4>{event.title}</h4>
+					<StyledInfoDiv>
+						<Calender />
+						<span>2nd Aug 2022 - 7th Aug 2022</span>
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx:32]: extract the date, location, duration etc

File: src/components/FeaturedEventCard.tsx [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+				{/* Right content */}
+				<StyledRightContent>
+					<StyledInfoDiv>
+						<People />
+						<span>12 particpants</span>
+						<StyledSpot>3 Spots left</StyledSpot>
+					</StyledInfoDiv>
+
+					<StyledPrice>
+						<p>{priceFormatter.format(10000)}</p>
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: nice!

Comment (Ozoemenamsolomon) [reply src/components/FeaturedEventCard.tsx]: but extract the price too

File: src/components/FeaturedEventCard.tsx [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+	event: EventDataType;
+}
+
+const FeaturedEventCard: React.FC<FeaturedEventProp> = ({ event }) => {
+	const [panelOpen, setpanelOpen] = useState(false);
+
+	return (
+		<StyledWrapperDiv>
+			<StyledDetailsSection>
+				{/* left content */}
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: remove this comment, comments won't be necessary if fuctions and variables are named properly

File: src/components/FeaturedEventCard.tsx [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+import Calender from '@assets/new/icons/Calender';
+import Time from '@assets/new/icons/Time';
+import Location from '@assets/new/icons/Location';
+import People from '@assets/new/icons/People';
+import { CTALink } from './CTA';
+import { priceFormatter } from './ProductCard';
+import ArrowDownUp from '@assets/new/icons/ArrowDownUp';
+import Call from '@assets/new/icons/Call';
+import Whatsapp from '@assets/new/icons/Whatsapp';
+// import People
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: !

File: src/components/FeaturedEventCard.tsx [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+	@media (min-width: 768px) {
+		padding: 2rem 2rem 1rem;
+	}
+
+	@media (min-width: 1028px) {
+		padding: 4rem 3rem 1rem;
+	}
+`;
+
+const StyledDetailsSection = styled.section`
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: use div, not section

File: src/components/styled/index.tsx [hunk 7]
```diff
@@ -62,3 +62,27 @@ export const Container = styled.div<{
 		}}
 	}
 `;
+
+interface StyledButtonProps {
+	white?: boolean;
+}
+
+export const StyledCustomButton = styled.button<StyledButtonProps>`
```
Comment (Ozoemenamsolomon) [inline src/components/styled/index.tsx]: What's this?

File: src/components/FeaturedEventCard.tsx (hunk 4 above)
Comment (Ozoemenamsolomon) [reply src/components/FeaturedEventCard.tsx]: All your styled components must not have styled prefix, they are redundant

File: src/components/ServiceStandard.tsx [hunk 8]
```diff
@@ -1,44 +1,34 @@
...
-				</p>
+				<h3>{heading}</h3>
+				<p>{paragraph}</p>
 			</ServiceWriteUp>
 			<StyledServiceIcon>
 				<ImageDescriptionWrapper>
-					{services.map((service, index) => (
-						<ImageDescription key={index}>
+					{servicesIcon.map((service, index) => (
+						<ImageDescription length={servicesIcon.length} key={index}>
```
Comment (Ozoemenamsolomon) [inline src/components/ServiceStandard.tsx]: add a discriminator to the key

File: src/components/FeaturedEventCard.tsx [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+
+	@media (min-width: 920px) {
+		gap: 2rem;
+		& > a > button {
+			font-size: 1rem;
+		}
+	}
+`;
+
+const StyledInfoSection = styled.section``;
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEventCard.tsx]: use div [check out](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

File: src/components/styled/Forms.tsx [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+export const StyledFormButtonControl = styled.div`
+  text-align: center;
+
+  & > button {
+    // display: block;
+    width: 100%;
+    font-size: 0.7rem;
+  }
+
+  @media(min-width: 768px){
```
Comment (Ozoemenamsolomon) [inline src/components/styled/Forms.tsx]: remove

File: src/components/FeaturedEvents.tsx [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+import FeaturedEventCard from './FeaturedEventCard';
+import { Container, StyledCustomButton } from './styled';
+
+const FeaturedEvents = () => {
+	const [displayElements, setdisplayElements] = useState(2);
+	const [increment] = useState(1);
+	// increment is how many more you add to the displayed list.
+	// displayElements is how many to render on mount.
+
+	const onClickLoad = () => {
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEvents.tsx]: make your variable names more decriptive, this function should be called `loadMoreEvent`, I guess that's what it's doing

File: src/components/FeaturedEvents.tsx [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+		margin-bottom: 5rem;
+	}
+`;
+
+const StyledLoadMore = styled.div`
+	text-align: center;
+	margin: 3rem 0;
+
+	& > button {
+		width: 16rem;
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEvents.tsx]: doesn't seem right 🤔

File: src/styles/globals.css [hunk 13]
```diff
@@ -82,13 +88,13 @@ img[object-fit='contain'] {
 
 /* This is the css section for swiper used in the home page */
 
-.swiper-pagination-bullet-active{
-  background: var(--oex-orange) !important;
+.swiper-pagination-bullet-active {
```
Comment (Ozoemenamsolomon) [inline src/styles/globals.css]: remove this and put it within the swiper wrapper

File: src/pages/contact.tsx [hunk 14]
```diff
@@ -1,161 +1,246 @@
...
+	const [formData, setformData] = useState({
+		fullname: '',
+		email: '',
+		phone: '',
+		message: '',
+	});
+
+	const { fullname, email, message, phone } = formData;
+
+	type InputChangeType =
```
Comment (Ozoemenamsolomon) [inline src/pages/contact.tsx]: type def outside component

File: src/pages/contact.tsx [hunk 15]
```diff
@@ -1,161 +1,246 @@
...
+										name="email"
+										onChange={onInputChange}
+										placeholder="Enter your email address here"
+										value={email}
+									/>
+								</StyledFormControl>
+								<StyledFormControl>
+									<label htmlFor="phone">Phone Number</label>
+									<input
+										type="text"
```
Comment (Ozoemenamsolomon) [inline src/pages/contact.tsx]: type="tel"

File: src/pages/contact.tsx [hunk 16]
```diff
@@ -1,161 +1,246 @@
...
 
 const Contact = () => {
+	const [formData, setformData] = useState({
+		fullname: '',
+		email: '',
+		phone: '',
+		message: '',
+	});
+
+	const { fullname, email, message, phone } = formData;
```
Comment (Ozoemenamsolomon) [inline src/pages/contact.tsx]: camelCase

File: src/data/eventsData.ts [hunk 17]
```diff
@@ -0,0 +1,107 @@
+
+
+export interface EventDataType   { 
+  id: number,
+  title: string,
+  event_format: string,
```
Comment (Ozoemenamsolomon) [inline src/data/eventsData.ts]: make this an enum type

File: src/data/eventsData.ts [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+  start_date: string,
```
Comment (Ozoemenamsolomon) [inline src/data/eventsData.ts]: `startDateTime` combine the date and time using ISO format, instead of having separate attribute for `startTime`

Comment (Ozoemenamsolomon) [reply src/data/eventsData.ts]: For example, "2007-04-05T14:30Z" or "2007-04-05T12:30−02:00".

File: src/data/eventsData.ts [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+  end_date: string,
+  start_time: string,
+  end_time: string,
+  location: {city: string, state: string},
+  price: number,
+  contact: {call: string, chat:string}, 
+  course_info: {
```
Comment (Ozoemenamsolomon) [inline src/data/eventsData.ts]: this should be of type string, a markdown string

File: src/data/openJobPositionsData.ts [hunk 20]
```diff
@@ -0,0 +1,29 @@
+export interface OpenJobPositionsType {
+	title: string;
+	description: string;
+	type: string;
```
Comment (Ozoemenamsolomon) [inline src/data/openJobPositionsData.ts:4]: of type enum

File: src/data/openJobPositionsData.ts [hunk 21]
```diff
@@ -0,0 +1,29 @@
...
+	location: string;
+	department: string;
+	applyLink: string;
```
Comment (Ozoemenamsolomon) [inline src/data/openJobPositionsData.ts:7]: you have to be consistent with your naming convention, here you are using camelCase which I find better but you're using another convention somewhere else

File: src/components/ServiceStandard.tsx (hunk 8 above)
Comment (Ozoemenamsolomon) [reply src/components/ServiceStandard.tsx]: use the ServiceCard here

File: src/components/ServiceStandard.tsx [hunk 22]
```diff
@@ -106,20 +95,22 @@ const StyledServiceIcon = styled.div`
...
 	& > p {
-		color: var(--text-colour-p);
+		color: var(--text-colour-grey);
+		width: 50%;
+		text-align: center;
 	}
 
 	@media (min-width: 1028px) {
-		flex: 33%;
+		width: ${({ length }) => (length > 5 ? '33%' : '50%')};
```
Comment (Ozoemenamsolomon) [inline src/components/ServiceStandard.tsx]: doesn't seem right

File: src/pages/careers.tsx [hunk 23]
```diff
@@ -0,0 +1,360 @@
...
+					'Do you love to be a part of a passionate team that enables human potential? We are building the best experience for you!'
+				}>
+				<SocialMediaButtons
+					height={20}
+					width={15}
+					color={'var(--oex-orange)'}
+				/>
+			</InfoText>
+
+			{/* Open positions */}
```
Comment (Ozoemenamsolomon) [inline src/pages/careers.tsx]: if you name your variables well you won't need comments

File: src/pages/careers.tsx [hunk 24]
```diff
@@ -0,0 +1,360 @@
...
+							fill
+							alt="career image"
+						/>
+					</StyledImageSmallDiv>
+				</StyledImageContentDiv>
+			</Container>
+
+			{/* Hiring process */}
+
+			<StyledHiringProcess>
```
Comment (Ozoemenamsolomon) [inline src/pages/careers.tsx:80]: this should be contained in the container above

File: src/components/shared/Address.tsx [hunk 25]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components';
+
+const Address = () => {
```
Comment (Ozoemenamsolomon) [inline src/components/shared/Address.tsx:4]: why shared? when you're just using it on contact page?

File: src/components/shared/HeaderParagraph.tsx [hunk 26]
```diff
@@ -0,0 +1,63 @@
...
+export interface HeaderParagraphType {
+	heading?: string;
+	paragraph?: string;
+}
+
+interface HeaderParagraphProps {
+	data: HeaderParagraphType;
+}
+
+const HeaderParagraph: React.FC<HeaderParagraphProps> = ({ data }) => {
```
Comment (Ozoemenamsolomon) [inline src/components/shared/HeaderParagraph.tsx:13]: shared?

File: src/components/JobPositonCard.tsx [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+			<CTALink clasname="btn" href={data.applyLink}>
+				Apply Now
+			</CTALink>
+		</StyledJobPositionCard>
+	);
+};
+
+export default JobPositonCard;
+
+export const StyledJobPositionCard = styled.div`
```
Comment (Ozoemenamsolomon) [inline src/components/JobPositonCard.tsx]: why are you exporting these?

Comment (Ozoemenamsolomon) [reply src/components/JobPositonCard.tsx]: why? when you're just using it on this file?

File: src/components/JobPositonCard.tsx [hunk 28]
```diff
@@ -0,0 +1,88 @@
...
+		width: 10px;
+		height: 10px;
+		-moz-border-radius: 7.5px;
+		-webkit-border-radius: 7.5px;
+		border-radius: 7.5px;
+		background-color: var(--oex-grey);
+		margin: 0rem 0.5rem;
+	}
+
+	@media (min-width: 768px) {
```
Comment (Ozoemenamsolomon) [inline src/components/JobPositonCard.tsx]: remove

File: src/components/JobPositonCard.tsx [hunk 29]
```diff
@@ -0,0 +1,88 @@
...
+			<StyledJobPositionCardContent>
+				<h3>{data.title}</h3>
+				<p>{data.description}</p>
+				<StyledContentDiv>
+					<span>{data.type}</span>
+					<span>{data.location}</span>
+					<span>{data.department}</span>
+				</StyledContentDiv>
+			</StyledJobPositionCardContent>
+			<CTALink clasname="btn" href={data.applyLink}>
```
Comment (Ozoemenamsolomon) [inline src/components/JobPositonCard.tsx]: className

File: src/components/ImageInfoHeader.tsx [hunk 30]
```diff
@@ -0,0 +1,101 @@
...
+		gap: 8rem;
+	}
+`;
+
+const StyledImageDiv = styled.div`
+	position: relative;
+	margin-bottom: 4rem;
+	overflow: hidden;
+	border-radius: 0.5rem;
+	width: 100%;
```
Comment (Ozoemenamsolomon) [inline src/components/ImageInfoHeader.tsx]: div is a block element

File: src/components/FeaturedEvents.tsx [hunk 31]
```diff
@@ -0,0 +1,72 @@
+import { featuredEvents } from '@data/eventsData';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import FeaturedEventCard from './FeaturedEventCard';
+import { Container, StyledCustomButton } from './styled';
+
+const FeaturedEvents = () => {
+	const [displayElements, setdisplayElements] = useState(2);
+	const [increment] = useState(1);
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEvents.tsx]: you don't need useState here, a simple variable will do

File: src/components/FeaturedEvents.tsx [hunk 32]
```diff
@@ -0,0 +1,72 @@
...
+const StyledWrapperDiv = styled.div`
+	background-color: var(--oex-off-white);
+	padding: 3rem 0rem;
+
+	& > button {
+		display: block;
+		margin: 3rem 0;
+	}
+
+	@media (min-width: 768px) {
```
Comment (Ozoemenamsolomon) [inline src/components/FeaturedEvents.tsx]: remove

File: src/components/SupportInfoCard.tsx [hunk 33]
```diff
@@ -0,0 +1,82 @@
...
+	return (
+		<StyledSupportInfoCard>
+			<StyledIconWrapper>
+				<CardIcon />
+			</StyledIconWrapper>
+			<h4>{cardTitle}</h4>
+			<p>{paragraph}</p>
+			<StyledBtnContainer>
+				{buttons.map(({ title, Icon, link }, index) => (
+					<Link key={index} href={link} passHref>
```
Comment (Ozoemenamsolomon) [inline src/components/SupportInfoCard.tsx]: discriminate

**Review by Ozoemenamsolomon — CHANGES_REQUESTED** [review]
Please look into these reviews and let us align on them later

File: src/components/FeaturedEventCard.tsx (hunk 1 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Done

File: src/components/FeaturedEventCard.tsx (hunk 3 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Done

File: src/components/FeaturedEventCard.tsx (hunk 4 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Ok Noted.

File: src/components/FeaturedEventCard.tsx (hunk 5 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Resolved

File: src/components/FeaturedEventCard.tsx (hunk 6 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Resolved

File: src/components/styled/index.tsx (hunk 7 above)
Comment (ebuka-nwabueze) [reply src/components/styled/index.tsx]: It was used for the initial design but has been replaced with CTA

File: src/components/ServiceStandard.tsx (hunk 8 above)
Comment (ebuka-nwabueze) [reply src/components/ServiceStandard.tsx]: Resolved

File: src/components/FeaturedEventCard.tsx (hunk 9 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx]: Resolved

File: src/components/styled/Forms.tsx (hunk 10 above)
Comment (ebuka-nwabueze) [reply src/components/styled/Forms.tsx]: Resolved

File: src/components/FeaturedEvents.tsx (hunk 11 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEvents.tsx]: Resolved.

File: src/components/FeaturedEvents.tsx (hunk 12 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEvents.tsx]: Changed the targeting to a class.

File: src/styles/globals.css (hunk 13 above)
Comment (ebuka-nwabueze) [reply src/styles/globals.css]: According to the documentation. The css resides in the global style sheet

File: src/pages/contact.tsx (hunk 14 above)
Comment (ebuka-nwabueze) [reply src/pages/contact.tsx]: Resolved

File: src/pages/contact.tsx (hunk 16 above)
Comment (ebuka-nwabueze) [reply src/pages/contact.tsx]: resolved

File: src/pages/contact.tsx (hunk 15 above)
Comment (ebuka-nwabueze) [reply src/pages/contact.tsx]: resolved

File: src/data/eventsData.ts (hunk 18 above)
Comment (ebuka-nwabueze) [reply src/data/eventsData.ts]: Resolved

File: src/data/eventsData.ts (hunk 17 above)
Comment (ebuka-nwabueze) [reply src/data/eventsData.ts]: I don't understand what you mean. 
Which of the property should be enum?
They are all different id, title and eventFormat

File: src/data/openJobPositionsData.ts (hunk 20 above)
Comment (ebuka-nwabueze) [reply src/data/openJobPositionsData.ts:4]: Same as the earlier comment. Not sure what you mean by enum here?

File: src/data/eventsData.ts (hunk 19 above)
Comment (ebuka-nwabueze) [reply src/data/eventsData.ts]: Not sure what you mean here?

File: src/data/openJobPositionsData.ts (hunk 21 above)
Comment (ebuka-nwabueze) [reply src/data/openJobPositionsData.ts:7]: All properties are now camelCase

File: src/components/ServiceStandard.tsx (hunk 22 above)
Comment (ebuka-nwabueze) [reply src/components/ServiceStandard.tsx]: resolved

File: src/pages/careers.tsx (hunk 24 above)
Comment (ebuka-nwabueze) [reply src/pages/careers.tsx:80]: It is not meant to be within the container. The containter has padding and it doesn't match with the design

File: src/components/shared/Address.tsx (hunk 25 above)
Comment (ebuka-nwabueze) [reply src/components/shared/Address.tsx:4]: I did it as it can be reusable

File: src/components/shared/HeaderParagraph.tsx (hunk 26 above)
Comment (ebuka-nwabueze) [reply src/components/shared/HeaderParagraph.tsx:13]: I created the folder to hold components that can be used in multiple places. The project is growing and folder structure is expanding.

File: src/components/JobPositonCard.tsx (hunk 27 above)
Comment (ebuka-nwabueze) [reply src/components/JobPositonCard.tsx]: Resolved

File: src/components/JobPositonCard.tsx (hunk 29 above)
Comment (ebuka-nwabueze) [reply src/components/JobPositonCard.tsx]: resolved

File: src/components/ImageInfoHeader.tsx (hunk 30 above)
Comment (ebuka-nwabueze) [reply src/components/ImageInfoHeader.tsx]: resolved

File: src/components/FeaturedEvents.tsx (hunk 32 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEvents.tsx]: resolved

File: src/components/SupportInfoCard.tsx (hunk 33 above)
Comment (ebuka-nwabueze) [reply src/components/SupportInfoCard.tsx]: resolved

File: src/components/FeaturedEvents.tsx (hunk 31 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEvents.tsx]: resolved

File: src/components/JobPositonCard.tsx (hunk 28 above)
Comment (ebuka-nwabueze) [reply src/components/JobPositonCard.tsx]: resolved

File: src/components/FeaturedEventCard.tsx (hunk 2 above)
Comment (ebuka-nwabueze) [reply src/components/FeaturedEventCard.tsx:32]: I will create a function for this.

**Review by Ozoemenamsolomon — APPROVED** [review]
looks good

**Pull request merged**